eslint.config.mjs

```diff
@@ -5,6 +5,8 @@ import { dirname } from 'path';
 const __filename = fileURLToPath(import.meta.url);
 const __dirname = dirname(__filename);
 
+const LEVEL = 'off';
```

Comment:
For experiments, Ok. But for production - this is harmful

Reply:
I've set it back to warn. These are rules that needs to be discussed anyway, because they are very hard to understand, and copilot doesn't seem to have fixes for them. Perhaps they are non-issues?